### DISCUSSION TITLE
Build a chrome extension

### DISCUSSION THREAD

**Discussion by AndonMitev** [vercel/next.js#68710] [Help] [answered]
### Summary

Hey i'm using `"next": "^14.2.5"`, and I have created:

manifest.json file
```
{
    "manifest_version": 3,
    "name": "Minimal Manifest",
    "version": "1.0.0",
    "description": "A basic example extension with only required keys",
    "icons": {
        "48": "images/icon-48.png",
        "128": "images/icon-128.png"
    }
}
```

then my build comand looks like this:

`"build": "next build && mv out/_next out/next && rsync -a --exclude='.*' extension/ out/",`

and in next.config.mjs i do have:

`output: "export"`,

the problem that i'm facing is that on build I do see this:

`Server Actions are not supported with static export.`

because of the `output: "export"`,, so do you have any example how to build chrome extension but using server actions too

**Comment by icyJoseph** [comment, marked as answer]
The problem is that Server Actions communicate with a server, and when you export, no server is created.

What is your server action doing?

**Reply by AndonMitev** [reply]
Am i do have some server action that do use supabase calls, fetching external API, a few cron jobs that are api routes actually, I feel like I'll need to go back in good old days with react and external server for this goal

**Reply by icyJoseph** [reply]
export is the solution, but a server action, is a way for the client to trigger a function on the server, without exposing its implementation details to the browser.
